plugins/Morpheus/vue/src/Demo/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: [
+    '../../../../../.eslintrc.js',
+  ],
+  rules: {
+    'max-len': 'off',
+  },
+}
```

Comment:
Is this file still needed? Doesn't look like there are any lines that are too long.

Reply:
It's gone now so I'm guessing it's not.